docs/src/content/common-props/common-props.md

```diff
@@ -226,6 +226,12 @@ class App extends React.Component {
 ReactDOM.render(<App/>, mountNode);
 ```
 
+## disableInlineStyles
+
+`type: boolean`
+
+The `disableInlineStyles` prop allows Victory components to work better with CSS classes or styled-components. By default, Victory provides inline styles to chart components, which will override any conflicting CSS classes. This flag will remove the inline styles, making it easier to provide custom styling for components.
+
```

Comment:
Since this prop is needed for multiple components, I added it to the common prop types. An example here might be helpful as well.